streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -554,6 +561,10 @@ private void runLoop() {
                                  "trigger a new rebalance.");
                     assignmentErrorCode.set(AssignorError.NONE.code());
                     mainConsumer.enforceRebalance();
+                } else if (nextProbingRebalanceMs.get() < System.currentTimeMillis()) {
+                    log.info("The probing rebalance interval has elapsed since the last rebalance, triggering a " +
+                                "rebalance to probe for newly aught-up clients");
```

Comment:
```suggestion
                                "rebalance to probe for newly caught-up clients");
```

"aught" is so 2000s.